akka-http-core/src/test/scala/akka/http/model/UriSpec.scala

```diff
@@ -406,7 +418,7 @@ class UriSpec extends WordSpec with Matchers {
       normalize("http://sourceforge.net/projects/uriparser/") shouldEqual "http://sourceforge.net/projects/uriparser/"
       normalize("http://sourceforge.net/project/platformdownload.php?group_id=182840") shouldEqual "http://sourceforge.net/project/platformdownload.php?group_id=182840"
       normalize("mailto:test@example.com") shouldEqual "mailto:test@example.com"
-      normalize("file:///bin/bash") shouldEqual "file:///bin/bash"
+      normalize("file:///bin/bash") shouldEqual "file:/bin/bash"
```

Comment:
That would probably settle it for now. For http it isn't really a question, it's more the question how to handle other schemes and for that I agree that revisiting the decision once use-cases come up would be good enough.